attest/verifier/types/src/convert/collateral.rs

```diff
@@ -138,12 +139,44 @@ fn version_3_1_empty_collateral() -> sgx_ql_qve_collateral_t {
     collateral
 }
 
+impl Digestible for prost::Collateral {
+    fn append_to_transcript<DT: DigestTranscript>(
+        &self,
+        context: &'static [u8],
+        transcript: &mut DT,
+    ) {
+        let typename = b"Collateral";
+        transcript.append_agg_header(context, typename);
+
+        let Self {
+            pck_crl_issuer_chain,
+            root_ca_crl,
+            pck_crl,
+            tcb_info_issuer_chain,
+            tcb_info,
+            qe_identity_issuer_chain,
+            qe_identity,
+        } = self;
+        pck_crl_issuer_chain.append_to_transcript(context, transcript);
+        root_ca_crl.append_to_transcript(context, transcript);
+        pck_crl.append_to_transcript(context, transcript);
+        tcb_info_issuer_chain.append_to_transcript(context, transcript);
+        tcb_info.append_to_transcript(context, transcript);
+        qe_identity_issuer_chain.append_to_transcript(context, transcript);
+        qe_identity.append_to_transcript(context, transcript);
```

Comment:
The context should be the field name
e.g. see https://github.com/mobilecoinfoundation/mobilecoin/blob/master/blockchain/types/src/block.rs#L188

Reply:
Looks like I missed this in previous reviews so that should also get addressed

Reply:
Good catch!

I've updated the implementation to derive `Digestible`, but kept the tests (with correct contexts).